io.github.pol_rivero.DesktopPlus.yaml

```diff
@@ -0,0 +1,115 @@
+    cleanup:
+      - "*"
+    sources:
+      - type: script
+        dest-filename: node-gyp
+        commands:
+          - node /usr/lib/sdk/node22/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js
+            $@
+  - name: github-desktop-plus
+    buildsystem: simple
+    build-options:
+      # Add the node bin directory & yarn directory.
+      append-path: /app/bin:/usr/lib/sdk/node22/bin:/run/build/github-desktop-plus/flatpak-node/chromedriver:/usr/lib/sdk/node22/lib/node_modules/npm/node_modules/node-gyp/bin
+      env:
+        # Sets the directory where Node is located so way npm won't download the headers.
+        npm_config_nodedir: /usr/lib/sdk/node22
+        # Set the Electron cache directory.
+        ELECTRON_CACHE: /run/build/github-desktop-plus/flatpak-node/electron-cache
+        # Skip Electron related downloads
+        ELECTRON_SKIP_BINARY_DOWNLOAD: "1"
+        OFFLINE: "1"
+        CHROMEDRIVER_SKIP_DOWNLOAD: "true"
+        # Use predownloaded dugite-native
+        TMPDIR: /run/build/github-desktop-plus/flatpak-node/tmp
+        XDG_CACHE_HOME: /run/build/github-desktop-plus/flatpak-node/cache
+    build-commands:
+      - HOME=$PWD node ./vendor/yarn-1.21.1.js config --offline set yarn-offline-mirror
+        $FLATPAK_BUILDER_BUILDDIR/flatpak-node/yarn-mirror
+      - ln -s $XDG_CACHE_HOME/node-gyp $HOME/.electron-gyp
+      - node ./vendor/yarn-1.21.1.js --offline --verbose
+      # Run electron-builder, passing the architecture arguments to it.
+      - . flatpak-node/electron-builder-arch-args.sh; node ./vendor/yarn-1.21.1.js
+        run --offline build:prod $ELECTRON_BUILDER_ARCH_ARGS
+      # Copy the resulting, unpacked directory to /app.
+      # (A glob is used because the directory name may contain the current arch.)
+      - cp -r dist/github-desktop-plus-linux-* /app/github-desktop-plus
+      # Install the wrapper script to start it.
+      - install -Dm755 start-github-desktop-plus.sh /app/bin/start-github-desktop-plus
+    post-install:
+      - install -D io.github.pol_rivero.DesktopPlus.desktop -t  /app/share/applications/
+      - install -D io.github.pol_rivero.DesktopPlus.metainfo.xml -t  /app/share/metainfo/
+      - install -Dm644 app/static/linux/logos/32x32.png  /app/share/icons/hicolor/32x32/apps/$FLATPAK_ID.png
+      - install -Dm644 app/static/linux/logos/64x64.png  /app/share/icons/hicolor/64x64/apps/$FLATPAK_ID.png
+      - install -Dm644 app/static/linux/logos/128x128.png  /app/share/icons/hicolor/128x128/apps/$FLATPAK_ID.png
+      - install -Dm644 app/static/linux/logos/256x256.png  /app/share/icons/hicolor/256x256/apps/$FLATPAK_ID.png
+      - install -Dm644 app/static/linux/logos/512x512.png  /app/share/icons/hicolor/512x512/apps/$FLATPAK_ID.png
+      # https://github.com/shiftkey/desktop/issues/21
+      - find "/app" -type f -executable -exec sed -i 's/libcurl-gnutls\.so\.4/libcurl.so.4\x00\x00\x00\x00\x00\x00\x00/g' {} \; 2>/dev/null || true
```

Comment:
what can of thing is that. Patching sonames ? Nope. Should be rebuilt.

Reply:
why do you want to change to `libcurl`? because `libcurl-gnutls` is not in the runtime? then the solution is to build it inside the app.